ray-curator/ray_curator/examples/audio/fleurs_and_wer_example.py

```diff
@@ -0,0 +1,71 @@
+from dataclasses import dataclass
+
+from loguru import logger
+from nemo.core.config import hydra_runner
+from omegaconf import OmegaConf
+
+from ray_curator.backends.xenna import XennaExecutor
+from ray_curator.pipeline import Pipeline
+from ray_curator.stages.audio.datasets.fleurs.create_initial_manifest import CreateInitialManifestFleursStage
+from ray_curator.stages.audio.io.asr_inference import AsrNemoInferenceStage
+from ray_curator.stages.resources import Resources
+
+
+@dataclass
+class TranscriptionConfig:
+    """
+    Transcription Configuration for audio to text transcription.
+    """
+
+    # Required configs
+    model_name: str  # NeMo model_name
+    raw_data_dir: str
+    lang: str | None = "hy_am"
+    split: str | None = "dev"
+
+
+def create_audio_pipeline(args: TranscriptionConfig) -> Pipeline:
+    # Define pipeline
+    pipeline = Pipeline(name="audio_inference", description="Inference audio and filter by WER threshold.")
+
+    # Add stages
+    # Add the composite stage that combines reading and downloading
+    pipeline.add_stage(
+        CreateInitialManifestFleursStage(
+            lang=args.lang,
+            split=args.split,
+            raw_data_dir=args.raw_data_dir,
+        )
+    )
+    pipeline.add_stage(
+        AsrNemoInferenceStage(model_name=args.model_name, batch_size=16, cuda="cuda", _resources=Resources(gpus=1.0))
```

Comment:
A good idea might be to do this instad:

`AsrNemoInferenceStage(model_name=args.model_name, cuda="cuda").with_(batch_size=16 _resources=Resources(gpus=1.0)))`